packages/frontend/src/app/views/sessions/login/login.component.scss

```diff
@@ -0,0 +1,20 @@
+:host {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer div {
```

Comment:
Let's make it bemish

Reply:
The other solution is to use:
display: flex;
align-items: center;
justify-content: space-between;